python-project/mortar/util.py

```diff
@@ -30,3 +30,10 @@ def mktemp(*args, **kwargs) -> str:
         (_, output_path) = mkstemp(*args, **kwargs)
 
     return output_path
+
+
+def get_file_text(file) -> str:
```

Comment:
This works fine, but there are more idiomatic approaches that can be used from sites that would call this function.

Using `with`:

```
with open(file) as fi:
    file_text = fi.read()
    # the file is automatically closed at the end of the "with" block

# use file_text now
```

Or using the `pathlib` module in the standard library (I just recently learned this way):

```
from pathlib import Path # imported at the top of the calling module

file_text = Path(file).read_text()
```